themes/startwords/README.md

```diff
@@ -1,31 +1,28 @@
-# Blank
+# <div align="center"><img src="https://startwords.cdh.princeton.edu/logotype.svg" /> Theme</div>
 
-Blank — starter [Hugo](https://gohugo.io/) theme for developers. Use it to make your own theme.
+[![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC%20BY--NC--SA%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)
+[![Hugo](https://img.shields.io/badge/hugo-0.67-blue.svg)](https://gohugo.io)
 
-[Live Demo](https://themes.gohugo.io/theme/blank/)
+The Startwords Hugo theme is designed for [a journal of the same name](https://startwords.cdh.princeton.edu/) by the [Center for Digital Humanities](cdh.princeton.edu/) at Princeton University.
 
-![Blank theme screenshot](https://github.com/Vimux/blank/blob/master/images/splash.png)
+## Goals
 
-## Installation
+The main architectural goals of this theme are to:
```

Comment:
I'm not sure all of these goals apply to the theme.. (e.g. the choice of static site generator applies to _Startwords_ more generally but not to the theme)

Reply:
agree that this one specifically is a goal for startwords but redundant here